packages/blockchain-api/test/blockscout.test.ts

```diff
@@ -934,4 +934,161 @@ describe('Blockscout', () => {
       ]
     `)
   })
+  it('should handle as escrow the meta-transactions coming from Komenci', async () => {
+    const result = await blockscoutAPI.getTokenTransactions({
+      address: '0x566ce6b765f038a98b5753ced0a65fd49aa0a07c',
+      token: 'cUSD',
+      localCurrencyCode: 'USD',
+    })
+
+    const transactions = result.reverse()
+    expect(transactions).toMatchInlineSnapshot(`
```

Comment:
I can't see that the result contains an event for block # 6203333 that matches the input you defined [here](https://github.com/celo-org/wallet/pull/276/commits/d1f833d077e8c3a0ff58b14eeaf47688a1ef59ae#diff-92da2c52d25b4d0dd3ead6bd5db36b01f0b44fa6447c101fca21424ca08906d2R592). Also, I believe you could just update the test case before that checks that the cUSD feed is correct as this new Escrow case should fall into that bucket.

Reply:
Ah, interesting, the fee is paid in cGLD, so that would be categorized as a cGLD transaction according to [this](https://github.com/celo-org/wallet/blob/main/packages/blockchain-api/src/transaction/Transaction.ts#L74).